egs2/TEMPLATE/speechlm1/speechlm.sh

```diff
@@ -0,0 +1,734 @@
+num_nodes=1          # The number of nodes.
+nj=32                # The number of parallel jobs.
+inference_nj=32      # The number of parallel jobs in decoding.
+gpu_inference=false  # Whether to perform gpu decoding.
+dumpdir=dump         # Directory to dump features.
+expdir=exp           # Directory to save experiments.
+python=python3       # Specify python to execute espnet commands.
+
+# Data preparation related
+local_data_opts=""  # Options to be passed to local/data.sh.
+data_tag=""         # You may combine the data in multiple ways. This is the tag for data composition 
+
+# Audio Feature extraction related
+feats_type=raw             # Input feature type.
+audio_format=flac          # Audio format: wav, flac, wav.ark, flac.ark  (only in feats_type=raw).
+min_wav_duration=0.1       # Minimum duration in second.
+max_wav_duration=30        # Maximum duration in second.
+fs=16000                   # Sampling rate.
+
+# Training related
+train_config=""    # Config for training.
+train_args=""      # Arguments for training, e.g., "--max_epoch 1".
+                   # Note that it will overwrite args in train config.
+tag=""             # Suffix for training directory.
+speechlm_exp=""         # Specify the directory path for experiment. If this option is specified, tag is ignored.
+speechlm_stats_dir=""   # Specify the directory path for statistics. If empty, automatically decided.
+num_splits=1       # Number of splitting for tts corpus.
+
+# Decoding related
+inference_config="" # Config for decoding.
+inference_args=""   # Arguments for decoding (e.g., "--threshold 0.75").
+                    # Note that it will overwrite args in inference config.
+inference_tag=""    # Suffix for decoding directory.
+inference_model=valid.acc.ave.pth # Model path for decoding.
+                                   # e.g.
+                                   # inference_model=train.loss.best.pth
+                                   # inference_model=3epoch.pth
+                                   # inference_model=valid.acc.best.pth
+                                   # inference_model=valid.loss.ave.pth
+vocoder_file=none  # Vocoder parameter file.
+download_model=""  # Download a model from Model Zoo and use it for decoding.
+
+# [Task dependent] Set the datadir name created by local/data.sh
+train_set=""     # Name of training set.
+valid_set=""     # Name of validation set used for monitoring/tuning network training.
+test_sets=""     # Names of test sets. Multiple items (e.g., both dev and eval sets) can be specified.
+task=            # when task is multi_task, skip data preparation and use train/valid yamls
+train_jsons=""
+valid_jsons=""
+test_jsons=""
+
+# Tokenization related
+oov="<unk>"         # Out of vocabrary symbol.
+blank="<blank>"     # CTC blank symbol.
+sos_eos="<sos/eos>" # sos and eos symbols.
+tokenization_choices=""
+codec_choice="EnCodec"
+codec_opts=""
+semantic_choice="WavLM"
+semantic_opts=""
```

Comment:
Do we use this part?

Reply:
This is reserved for the future when we do LM-based ASR etc. Currently, it's not in use.